.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
```

Comment:
_(arbitrary comment location to start a thread)_

In DSTestPlus, we should be able to trim it down to this (fewer imports, one less contract we inherit from)

```solidity
import "forge-std/Test.sol";
import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
import "solmate/test/utils/mocks/MockERC20.sol";

contract DSTestPlus is Test {
  using stdStorage for StdStorage;

  function setToll(address umbraAddr, uint256 newToll) public {
    stdstore.target(address(umbraAddr)).sig("toll()").checked_write(newToll);
  }
}
```